types/selenium-webdriver/lib/virtual_authenticator.d.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * Protocol for virtual authenticators
+ * @enum {string}
+ */
+export interface IProtocol {
+    CTAP2: string;
+    U2F: string;
+}
+
+/**
+ * AuthenticatorTransport values
+ * @enum {string}
+ */
+export interface ITransport {
+    BLE: string;
+    USB: string;
+    NFC: string;
+    INTERNAL: string;
+}
+
+export const Protocol: IProtocol;
+export const Transport: ITransport;
+
+/**
+ * Options for the creation of virtual authenticators.
+ * @see http://w3c.github.io/webauthn/#sctn-automation
+ */
+export class VirtualAuthenticatorOptions {
+    constructor();
+
+    getProtocol(): string;
+
+    setProtocol(protocol: string): void;
+
+    getTransport(): string;
+
+    setTransport(transport: string): void;
+
+    getHasResidentKey(): boolean;
+
+    setHasResidentKey(value: boolean): void;
+
+    getHasUserVerification(): boolean;
+
+    setHasUserVerification(value: boolean): void;
+
+    getIsUserConsenting(): boolean;
+
+    setIsUserConsenting(value: boolean): void;
+
+    getIsUserVerified(): boolean;
+
+    setIsUserVerified(value: boolean): void;
+
+    toDict(): Object;
+}
+
+/**
+ * A credential stored in a virtual authenticator.
+ * @see https://w3c.github.io/webauthn/#credential-parameters
+ */
+export class Credential {
+    constructor(
+        credentialId: Uint8Array,
+        isResidentCredential: boolean,
+        rpId: string,
+        userHandle: Uint8Array,
+        privateKey: string,
+        signCount: number
+    )
+
+    id(): Uint8Array;
+
+    isResidentCredential(): boolean;
+
+    rpId(): string;
+
+    userHandle(): Uint8Array;
+
+    privateKey(): string;
+
+    signCount(): number;
+
+    /**
+     * Creates a resident (i.e. stateless) credential.
+     * @param id Unique base64 encoded string.
+     * @param rpId Relying party identifier.
+     * @param userHandle userHandle associated to the credential. Must be Base64 encoded string.
+     * @param privateKey Base64 encoded PKCS
+     * @param signCount initial value for a signature counter.
+     * @returns A resident credential
+     */
+    static createResidentCredential(id: Uint8Array, rpId: string, userHandle: Uint8Array, privateKey: string, signCount: number): Credential;
+
+    /**
+     * Creates a non-resident (i.e. stateless) credential.
+     * @param id Unique base64 encoded string.
+     * @param rpId Relying party identifier.
+     * @param privateKey Base64 encoded PKCS
+     * @param signCount initial value for a signature counter.
+     * @returns A non-resident credential
+     */
+    static createNonResidentCredential(id: Uint8Array, rpId: string, privateKey: string, signCount: number): Credential;
```

Comment:
```suggestion
    createNonResidentCredential(id: Uint8Array, rpId: string, privateKey: string, signCount: number): Credential;
```
This method isn't static, [the tests](https://github.com/SeleniumHQ/selenium/blob/000c2cf10d522a90bb765e6ff6882114eefcadc4/javascript/node/selenium-webdriver/test/lib/credentials_test.js#L108) assert that.

Reply:
This method has been made static now with [MR](https://github.com/SeleniumHQ/selenium/pull/11182/files). The [tests](https://github.com/TamsilAmani/selenium/blob/d7453d1bd0a5cbdc7fea346433dc1ecd589e974c/javascript/node/selenium-webdriver/test/lib/credentials_test.js#L106) have also been modified.